assets/css/index.css

```diff
@@ -1,10 +1,44 @@
-#ProjectView {
-	width: 100%;
-
+body { 
+	height:100%;
+	background-color: rgba(125, 212, 250, 1);	
 }
 
 .row {
-	margin:4px;
-	padding: 8px;
 	min-height:40px;
+}
+
+#HistoryView {
+	margin-top: 50px; /* because of the header */ 
+	height:10%;
+	min-height: 60px;
+	padding:4px;
+	background-color: rgba(125, 212, 250, 1);
+}
+
+#HistorySketchesView {
+	overflow-x:scroll; 
```

Comment:
my reasoning is that you wouldn't want to lose undo control over versions that no longer fit in the window. to be able to see all the versions would be easiest while coding, but when we actually start cleaning up the interface we can see what works better.